gitpod-io/gitpod
# [server] Hook up usage API (4/5)

## Description

Configures `server` to consume the actual `UsageService` over gRPC.

## Related Issue(s)

Context #10325 

## How to test
 - create a team with billing
 - go to team -> Billed Usage and see that empty list

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:
